dist/images/ovndb-raft-functions.sh

```diff
@@ -29,9 +29,9 @@ ready_to_join_cluster() {
   if [[ $? != 0 ]]; then
     return 1
   fi
-  target=$(ovn-${db}ctl --db=${transport}:${init_ip}:${port} ${ovndb_ctl_ssl_opts} --data=bare --no-headings \
```

Comment:
In the commit message:
Addresses issue #977.

Instead of that do https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords#about-issue-references

so issue will get closed automatically with correct linkage